rkt/pods.go

```diff
@@ -731,16 +733,85 @@ func (p *pod) getState() string {
 	return state
 }
 
+// Returns the pid of the child, or 0 if not ready
+func getChildPID(ppid int) (int, error) {
```

Comment:
nit:
Overloading the pid return value to indicate when the child can't be read (pid of 0) would be normal in C, but in Go you should probably be returning a specific error.  Then the caller will be self-documenting in specifically handling that error.